src/component/numbers/Numbers.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import './numbers.scss';
+
+const numList = (from, to, type) => {
+  let arr = [];
+
+  for (let i = from; i <= to; i++) {
+    if (type) {
+      if (type == 'odd') {
+        if (i % 2 === 0) {
+          arr.push(i);
```

Comment:
After this operation you don't need to check next if statement 'if (type == 'even')'. Add here 'continue'

Reply:
changed the render logic, please take a look